### ISSUE TITLE
Add compile instructions to ReadMe

### ISSUE THREAD

**Issue by DanRStevens** [OutpostUniverse/OPHD#737]
We should add compile instructions to the ReadMe file to help with first time setup for new developers. The ReadMe is probably the most obvious place to look for such info. This should help new developers, and the curious, as they won't have to hunt for information. It's also the easiest to maintain as it's right in the repository, so is more likely to be updated if build instructions change.

We should also look at removing the Wiki article so we don't end up with duplicated, or worse, outdated information. This has been a bit of a problem for us on a few occasions, where outdated documentation has mislead new users. I'm not sure I even have write access to the Wiki anymore. Plus I don't really want to have to maintain yet another set of credentials, nor learn yet a new formatting language. It would be easier to maintain if everything was in one place (on GitHub), and if documentation was in one format (Markdown). Less confusion for users, less hassle for us.

----

In terms of updated instructions, we should mention the `vcpkg integrate install` step. We can dump mention of NuGet, which is no longer used.

If the instructions get to be too long, we can break things out into separate Markdown files, and link to them from the main ReadMe. Relative paths within the repository should work fine as hyperlinks on GitHub, and will be clickable. Two main sections will be development setup, and how to play.

----

Related: #736
Wiki article: https://wiki.outpost2.net/doku.php?id=outposthd:building_from_source&s[]=ophd

**Comment by ldicker83** [comment]
The Wiki is pretty updated though I did remove the NuGet mention. We can add these to the readme and link to forums for additional support. There are a few assumptions that are made -- namely that git is properly installed and that the user knows how to open a command prompt with administrator privs. Me thinks if a user doesn't know how to do that they probably have no business trying to build software from source and leave it to the wiki page/forums for additional support.

**Comment by ldicker83** [comment]
Okay, so an update.

I set up a VM with a clean install of Windows 10. There are a couple of steps missed in the Wiki guide about setting up vcpkg that causes the build to fail, specifically adding vcpkg to the path environment variable. This is critical or the build just plain fails.

As for debugging, it seems to work fine finding data. It's sort of running at this point -- the VM has no audio driver/hardware for whatever reason so loading music files causes a music load fail with the error "Unrecognized audio format" -- this I think is a little misleading and has more to do with NAS2D not handling no audio hardware gracefully (it hasn't really come up until now and the likelihood of this issue coming up on real hardware is slim... still, this shouldn't prevent the game from otherwise running). At least with sound files it states that it can't be loaded because the audio device hasn't been opened. Assuming a NULL mixer has been loaded this really shouldn't be throwing. Gonna have to investigate that.

Anyway, I did notice a lot of smearing and overall very poor visuals though I bet this has to do with the SuperVGA driver being used in VirtualBox. Again, I don't think this will be an issue on real hardware. I've tested it on machines as old as 2006 builds with AMD graphics drivers and it still works so this is probably just an artefact of doing this on a virtual machine.

TL;DR

There's the step of setting up the path environment variable for vcpkg to properly run which is critical to the build process running. Without this it's a chore. Will update the documentation and hopefully this will resolve most issues. Will need to verify on different production setups that aren't clean-room type of deals to ensure that other potential steps aren't being missed.

---

Screenies!

So... with this awful driver non-power of 2 textures just don't load (e.g., the background graphics). Loaded for the main menu screen, go figure, but not anything else. So lots of empty buffer fun going on.

And just... ugh. The whole game looks like it's through the lens of a camera with petroleum jelly smeared all over the lens. This is... this is bad. lol. BUT IT WORKS! :D

![ss10](https://user-images.githubusercontent.com/13858902/103434159-5f0c0680-4bcb-11eb-9429-19855f60b5d5.png)
![ss11](https://user-images.githubusercontent.com/13858902/103434223-549e3c80-4bcc-11eb-9bd2-aa8fa104e66c.png)
![ss12](https://user-images.githubusercontent.com/13858902/103434292-48ff4580-4bcd-11eb-9e36-afc63ee9dc14.png)

**Comment by DanRStevens** [comment]
That's useful information.

The sound issue sounds similar to what was happening when the mixer related DLLs weren't being copied from the vcpkg directory. Are you sure that's a VM issue? See #219 and #383.

I wonder if there is a way we could search for `vcpkg` in a default location in case it's not on the path.

**Comment by ldicker83** [comment]
It's not so much a VM issue as it is a "This hardware configuration has no audio output device so I'm going to barf now" issue. It's definitely not the DLL's not being copied because they are actually getting copied. When I fixed the VM with a proper audio interface the barfing stopped and it worked normally. I'll also point out that switching from VMWare's driver to the VirtualBox driver fixed the visual issues as well (minus no support for non-power of 2 textures, mostly background images).

**Comment by DanRStevens** [comment]
I'm going to close this case as it's now basically a somewhat off topic duplicate of the new Issue #878.
